src/gogolica/core/generate.clj

```diff
@@ -1,14 +1,13 @@
-(ns gogolica.generate
+(ns gogolica.core.generate
   "A namespace for functions that do most of the actual codegen of gogolica."
   (:gen-class)
   (:require [cheshire.core :as json]
             [clojure.string :as str]
             [clojure.set :as set]
             [me.raynes.fs :as fs]
             [clj-http.client :as http]
-            [gogolica.auth :as auth]
-            [gogolica.common :as common]
-            [gogolica.model :as model]
+            [gogolica.core.auth :as auth]
+            [gogolica.core.common :as common]
```

Comment:
In here we need `model` but we don't need those two

Reply:
It wasn't really needed, but it now is (61685ec)